# Question 3
-----

In [1]:
mean = (1/80) * 150
print("E(X) = " + str(mean))

E(X) = 1.875


$$X \sim Po(1.875)$$

Poisson is appropriate because n > 50 and p < 0.1

In [2]:
import math

class poisson:
    def __init__(self, lamb):
        self.lamb = lamb

    def prob(self, x):
        pois = math.exp(-self.lamb) * (pow(self.lamb, x) / math.factorial(x))
        return pois

    def freq(self, event, sample):
        freq = self.prob(event) * sample 
        freq = round(freq, 0)
        return freq

In [3]:
pois_dist = poisson(1.875)
PXE0 = pois_dist.prob(0)
PXE1 = pois_dist.prob(1)
PXGE2 = 1 - (PXE0 + PXE1)
print("P(X >= 2) = " + str(PXGE2))

P(X >= 2) = 0.5591044703208307


In [4]:
p = 1/80
q = 1 - p
print("q = " + str(q))

q = 0.9875


$$1 - \frac{n!}{0!(n - 0)!} * q^{n-0} * p^0 \ge 0.99$$

$$q^n \le 0.01$$

$$n \le \frac{\log 0.01}{\log 0.9875}$$

$$n \le 366.106$$

Hence, n must be at least 367